### PR TITLE
MPGS module v1.1

### PR DESCRIPTION
**Pull request by OnTapKristjan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,71 @@
-Mastercard Payment Gateway Services module for WooCommerce
```
Comment (jk00000) [inline README.md]: Why has the title been removed? A title is required. Please also list below the title which versions of WooCommerce are supported. Look at the prestashop module readme.md: https://github.com/Mastercard-Gateway/gateway-prestashop-module

File: i18n/mastercard-es_ES.po [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+#: includes/class-gateway.php:824
+msgid "For more information please contact your payment service provider."
+msgstr ""
+
+#: includes/class-gateway.php:829
+msgid "Checkout Interaction"
+msgstr ""
+
+#: includes/class-gateway.php:832
+msgid "Redirect Payment Page"
```
Comment (jk00000) [inline i18n/mastercard-es_ES.po]: Redirect to Payment Page

File: i18n/mastercard-es_ES.po [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+"Learn how to access your <a href=\"%s\" target=\"_blank\">Gateway API "
+"Credentials</a>."
+msgstr ""
+
+#: includes/class-gateway.php:861
+msgid "Test Sandbox"
+msgstr ""
+
+#: includes/class-gateway.php:862
+msgid "Enable test simulator sandbox mode"
```
Comment (jk00000) [inline i18n/mastercard-es_ES.po]: Enable test sandbox mode

File: i18n/mastercard-es_ES.po [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+msgid ""
+"Accept payments on your WooCommerce store using Mastercard Payment Gateway "
+"Services."
+msgstr ""
+"Acepta pagos en tu tienda WooCommerce usando Mastercard Payment Gateway "
+"Services."
+
+#: includes/class-gateway.php:196
+#, php-format
+msgid "Error communicating with Mastercard API: \"%s\""
```
Comment (jk00000) [inline i18n/mastercard-es_ES.po]: change to "Error communicating with payment gateway API:"

File: includes/class-gateway.php [hunk 5]
```diff
@@ -634,123 +757,133 @@ public function init_form_fields() {
...
 				'type'        => 'checkbox',
-				'description' => __( 'Place the payment gateway in sandbox mode using sandbox API keys (real payments will not be taken).', 'woocommerce' ),
+				'description' => __( 'Place the payment gateway in test mode using test API credentials (real payments will not be taken).',
+					'mastercard' ),
 				'default'     => 'yes'
 			),
 			'sandbox_username'   => array(
-				'title'       => __( 'Sandbox Merchant ID', 'woocommerce' ),
-				'type'        => 'text',
-				'description' => __( 'Get your API keys from your Mastercard account: Settings > API Keys.', 'woocommerce' ),
```
Comment (jk00000) [inline includes/class-gateway.php]: Can we add a description "this is your test merchant profile ID prefixed with TEST"

File: includes/class-payment-gateway-cc.php [hunk 6]
```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Copyright (c) 2019 Mastercard
```
Comment (jk00000) [inline includes/class-payment-gateway-cc.php]: Can we update copyright year to 2019 - 2020

**Comment by OnTapKristjan** [comment]
Pushed new changes that fix all issues above

File: README.md [hunk 7]
```diff
@@ -1,3 +1,72 @@
...
 
-Installation instructions: http://wiki.ontapgroup.com/display/MPGS/Installation+of+Module
+The module has been tested with the WordPress version:
+- Version 5.2.4
+- Version 5.3.2
+
+and with PHP versions:
+- 7.2
+- 7.3
+
```
Comment (jk00000) [inline README.md]: Please add:
This module shares the following customer information with the gateway:
- Billing address
- Shipping address
- Contact name
- Contact phone number
- Contact email address
- (optional) Cart line items

**Pull request merged**